examples/python/linear_regression.py

```diff
@@ -41,6 +41,6 @@ def loss_fn(w):
 throughput = num_iters / (toc - tic)
 
 print(
-    f"Loss {loss.item():.5f}, |w-w*| = {error_norm:.5f}, "
+    f"Loss {loss.item():.5f}, Mean Squared Eror : |w-w*| = {error_norm:.5f}, "
```

Comment:
`Eror` -> `Error`?

Reply:
It's not really an MSE (that would impliy a mean). It's really the L2 distance between $w$ and $w^*$. Maybe you could change to.

You could have `Norm of difference :` if you want to add something there for more helpful outputs.